dependencies/pom.xml

```diff
@@ -105,6 +105,8 @@
         <version.lib.micronaut.sql>4.4.0</version.lib.micronaut.sql>
         <!-- FIXME upgrade to 3.1 when it is released in Maven -->
         <version.lib.microprofile-config>3.0.1</version.lib.microprofile-config>
+        <version.lib.microprofile-cdi-tck>4.0.10</version.lib.microprofile-cdi-tck>
```

Comment:
TCK dependency still here, should be in root `pom.xml`

Reply:
My bad, I moved only one before. Now every TCK property and dependency is moved to `helidon/pom.xml`